css/style.css

```diff
@@ -1,4 +1,942 @@
+  width: 22px;
+  height: 3px;
+  background-color: #e51937;
+}
+
+.first-ln {
+  transform: translate(-7px, 7px) rotate(45deg);
+  top: 7px;
+}
+
+.second-ln {
+  transform: translate(7px, -7px) rotate(-45deg);
+  top: 21px;
+}
+
+.arrow:hover {
+  cursor: pointer;
+}
+
+/* Описание стилей Третьей и Четвертой секции */
+
+.wrap-information {
+  position: relative;
+}
+
+.info__title {
+  position: relative;
+  z-index: 2;
+  color: #e51937;
+  font-size: 71px;
+  font-weight: 600;
+  transform: translate(77px, 20px);
+}
+
+.info__img {
+  width: 1273px;
+  height: 843px;
+  position: absolute;
+  transform: translate(0, -128px);
+}
+
+.red-first {
+  position: absolute;
+  transform: translate(226%, -185px);
+  background-color: #e51937;
+  color: #fff;
+  max-width: 441px;
+  height: 900px;
+  box-sizing: border-box;
+  padding: 54px 0 0 55px;
+}
+
+.description__title {
+  margin: 0 auto;
+  font-weight: 500;
+  font-size: 31px;
+  text-align: left;
+  white-space: nowrap;
+  letter-spacing: 2px;
+}
+
+.description__text {
+  margin: 0 auto;
+  display: block;
+  max-width: 100%;
+  font-size: 16px;
+  padding: 11px 47px 71px 0;
+  letter-spacing: 1px;
+}
+
+.description__screen {
+  display: flex;
+  padding-bottom: 51px;
+}
+
+.displ {
+  display: flex;
+  flex-wrap: nowrap;
+}
+
+.data__display {
+  white-space: nowrap;
+  display: inline-block;
+}
+
+.description__wifi {
+  padding-bottom: 38px;
+}
+
+@media (min-width: 1200px) and (max-width: 1295px) {
+  .menubar__link:nth-child(5) {
+    margin-right: 170px;
+  }
+
+  .tesla-bgd {
+    height: auto;
+    width: 100%;
+    max-width: 1295px;
+    top: -103px;
+    left: 0;
+  }
+
+  .tesla-car {
+    padding: 23px 0 20px 107px;
+  }
+
+  .content__data-car {
+    padding: 0 232px 22px 199px;
+  }
+
+  .data__speed-block {
+    padding-right: 55px;
+  }
+}
+
+@media (min-width: 1024px) and (max-width: 1199px) {
+  .menubar__link:nth-child(5) {
+    margin-right: 120px;
+  }
+
+  .main-tesla {
+    display: flex;
+    justify-content: center;
+  }
+  .tesla-bgd {
+    height: auto;
+    width: 100%;
+    max-width: 1199px;
+    top: -75px;
+    left: 0;
+  }
+  .tesla-car {
+    padding: 65px 0 20px 4px;
+  }
+  .content__data-car {
+    padding: 0 calc(321px - 2 * (321px - (321 * 100% / 1440))) 22px;
+  }
+  .data__speed-block {
+    padding-right: 35px;
+  }
+
+  .test-btn {
+    padding-right: 0;
+  }
+
+  .testdrive {
+    margin-right: 60px;
+  }
+}
+
+@media (min-width: 900px) and (max-width: 1023px) {
+  .menubar__link {
+    display: none;
+  }
+  .tesla-bgd {
+    top: -35px;
+    left: 0;
+    width: 100%;
+    height: auto;
+    max-width: 1023px;
+  }
+  .tesla-car {
+    padding: 54px 0 20px 0;
+    width: 100%;
+  }
+  .content__data-car {
+    padding: 0 calc(321px - 2 * (321px - (321 * 100% / 1440))) 50px;
+    flex-direction: column;
+  }
+
+  .content__facts {
+    padding-bottom: 65px;
+  }
+  .data__speed-block {
+    padding-right: 104px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)